lib/wor/paginate/paginate.rb

```diff
@@ -2,37 +2,27 @@
 
 module Wor
   module Paginate
-    # The order of this array is important!
-    # In a future release we'll provide an interface to manipulate it
-    ADAPTERS = [
-      Adapters::KaminariAlreadyPaginated,
-      Adapters::WillPaginateAlreadyPaginated,
-      Adapters::WillPaginate,
-      Adapters::Kaminari,
-      Adapters::ActiveRecord,
-      Adapters::Enumerable
-    ].freeze
-
     def render_paginated(content, options = {})
       return render_paginate_with_include(content, options) if includes?(options)
 
       render json: paginate(content, options)
     end
 
+    # rubocop: disable Metrics/AbcSize
```

Comment:
:eye:

Reply:
I know it's not the cutest thing in the world, but you should see how this gets uglier and unreadable as you try to get rid of the `Metrics/AbcSize` rubocop warning. I don't know what is worst, but I guarantee it is not easy to avoid the warning. Maybe changing too much the `paginate()` method implementation